R/utils.R

```diff
@@ -967,6 +968,16 @@ validate_length_one <- function(x, name) {
   }
 }
 
+column_classes_are_valid <- function(data, columns, valid_classes) {
```

Comment:
I have some issues with this implementation.

1. Each column of the data frame could have multiple classes, e.g. `data.frame(x = structure(class = c("num", "numeric"), 1:10))`
2. I think each of the columns should have at least one match with `valid_classes`, like `any(class(x) %in% valid_classes)` or something

Reply:
Thanks. I made the change to match your suggestion.